README.md

```diff
@@ -90,10 +90,19 @@ export default Ember.Component.extend({
     }];
   }),
 
+  table: computed('model.[]', function() {
```

Comment:
Computed properties are not supposed to have side effects. If you're going to do this, then you might as well use an observer.